src/olympia/addons/models.py

```diff
@@ -1218,7 +1218,7 @@ def can_be_deleted(self):
         return not self.is_deleted
 
     def has_listed_versions(self):
-        return self.versions.filter(
+        return self.current_version or self.versions.filter(
```

Comment:
Doesn't look like much, but it's one (or more) extra query avoided on every page in devhub, in some cases the detail API, and many reviewer pages.